humble/distribution.yaml

```diff
@@ -3171,6 +3171,20 @@ repositories:
       type: git
       url: https://github.com/ros4hri/hri_actions_msgs.git
       version: humble-devel
+  hri_face_detect:
+    doc:
+      type: git
+      url: https://github.com/ros4hri/hri_face_detect.git
+      version: humble-devel
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/ros4hri/hri_face_detect-release.git
+      version: 2.0.10-2
```

Comment:
This stanza should be removed since the build farm binary packaging jobs will not succeed without metadata.